team-telnyx/webrtc
# chore(WEBRTC-295): remove dead code

-  remove dead code

## 📝 To Do


## ✋ Manual testing

1. Navigate into `packages/js`
2. Run `npm run build` && `npm test`
3. You should get green on both commands.
4. Run `npm pack` and get the `.tgz` file and run `npm install {file-name}.tgz` on `js/examples/react-video`
5. run `npm start` on `js/examples/react-video`
6. Make a call and receive a call and see if everything is working.
7. Open the `Console` tab from the browser and see if has no errors.
8. Open the `Network` tab, filter by `WS`, and click on `wss://rtc.telnyx.com:14938/` see the messages and check if has no errors.
9. You can install this lib `npm i -D ts-unused-exports` on `packages/js`  and run `./node_modules/.bin/ts-unused-exports ./tsconfig.json` to see all the unused code.

PS: This command will return a list of classes and files that are used on `react-native SDK`, `test.ts`, and helper functions.

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |![image](https://user-images.githubusercontent.com/16343871/102640168-d4da9180-4138-11eb-800c-deed397834f4.png)|

## ts-unused-exports - the result after removing the dead code.

![image](https://user-images.githubusercontent.com/16343871/102646985-c180f380-4143-11eb-8df0-3c7dc3b2b2d6.png)
